src/error.rs

```diff
@@ -35,182 +36,343 @@ use std::num::ParseIntError;
+    /// Curve not implemented
     CurveNotImplemented(String),
+    /// JWT key not found
     MissingKey,
+    /// Missing private key parametern JWK
     MissingPrivateKey,
+    /// Missing modulus in RSA key
     MissingModulus,
+    /// Missing modulus in RSA key
     MissingExponent,
+    /// Missing prime factor in RSA key
     MissingPrime,
+    /// Verifiable credential not found in JWT
     MissingCredential,
+    /// Verifiable presentation not found in JWT
     MissingPresentation,
+    /// JWT key parameters not found
     MissingKeyParameters,
+    /// Missing proof property
     MissingProof,
+    /// Missing issuance date
     MissingIssuanceDate,
+    /// Missing type VerifiableCredential
     MissingTypeVerifiableCredential,
+    /// Missing type VerifiablePresentation
     MissingTypeVerifiablePresentation,
+    /// Missing issuer property
     MissingIssuer,
+    /// Missing account id
     MissingAccountId,
+    /// Missing verificationMethod
     MissingVerificationMethod,
+    /// Missing verification relationship
     MissingVerificationRelationship(String, crate::vc::ProofPurpose, String),
+    /// Problem with JWT key
     Key,
+    /// Problem parsing Secp256k1 key
     Secp256k1Parse(String),
+    /// Problem parsing Secp256r1 key
     Secp256r1Parse(String),
+    /// A verification method MUST NOT contain multiple verification material properties for the same material. (DID Core)
     MultipleKeyMaterial,
+    /// Unable to convert date/time
     TimeError,
+    /// Invalid URI
     URI,
+    /// Invalid DID URL
     DIDURL,
+    /// Unable to dereference DID URL
     DIDURLDereference(String),
+    /// Unexpected DID fragment
     UnexpectedDIDFragment,
+    /// Invalid context
     InvalidContext,
+    /// DID controller limit exceeded
     ControllerLimit,
+    /// Missing context
     MissingContext,
+    /// Missing document ID
     MissingDocumentId,
+    /// Missing JWS in proof
     MissingProofSignature,
+    /// Expired proof
     ExpiredProof,
+    /// Proof creation time is in the future
     FutureProof,
+    /// Invalid proof domain
     InvalidProofPurpose,
+    /// Missing proof purpose
     MissingProofPurpose,
+    /// Invalid proof purpose
     InvalidProofDomain,
+    /// Invalid Signature
     InvalidSignature,
+    /// Signature length did not match expected length.
     UnexpectedSignatureLength(usize, usize),
+    /// Invalid JWS
     InvalidJWS,
+    /// Missing JWS Header
     MissingJWSHeader,
+    /// Missing credential schema for ZKP
     MissingCredentialSchema,
+    /// Unsupported property for linked data proof
     UnsupportedProperty,
+    /// Unsupported key type
     UnsupportedKeyType,
+    /// Unsupported type for linked data proof
     UnsupportedType,
+    /// Unsupported proof purpose
     UnsupportedProofPurpose,
+    /// Unsupported check
     UnsupportedCheck,
+    /// Blank node identifier in predicate is unsupported
     UnsupportedBlankPredicate,
-    TooManyBlankNodes,
```

Comment:
Just checking, was this removed by mistake on because of the ongoing LD-Framing research?

Reply:
Removed since it's old dead code (not used anywhere for a long time). If this would block 0.3.1, it can be removed from this PR.